core/controllers/admin.py

```diff
@@ -1226,6 +1270,115 @@ def _generate_dummy_classroom(self) -> None:
         else:
             raise Exception('Cannot generate dummy classroom in production.')
 
+    def _generate_dummy_suggestion_questions(
+            self, skill_id: str,
+            num_dummy_suggestion_ques_generate: int) -> None:
+        """Generates and loads the database with a specified number of
+            suggestion question for the selected skill.
+
+        Raises:
+            Exception. Cannot load suggestion questions in production mode.
+            Exception. User does not have enough rights to generate data.
+        """
+        assert self.user_id is not None
+        if constants.DEV_MODE:
+            if (feconf.ROLE_ID_QUESTION_ADMIN not in self.user.roles
+                or not user_services.can_submit_question_suggestions(
+                    self.user_id)):
+                raise Exception(
+                    'User does not have enough rights to generate data.')
+            for _ in range(num_dummy_suggestion_ques_generate):
+                content_id_generator = translation_domain.ContentIdGenerator()
+                content_id_generator.generate(
+                    translation_domain.ContentType.CONTENT)
+                content_id_generator.generate(
+                    translation_domain.ContentType.DEFAULT_OUTCOME)
+                state = state_domain.State.create_default_state(
+                'default_state',
+                content_id_generator.generate(
+                    translation_domain.ContentType.CONTENT),
+                content_id_generator.generate(
+                    translation_domain.ContentType.DEFAULT_OUTCOME),
+                is_initial_state=True)
+                state.update_interaction_id('TextInput')
+                solution_dict: state_domain.SolutionDict = {
+                    'answer_is_exclusive': False,
+                    'correct_answer': 'Solution',
+                    'explanation': {
+                        'content_id': content_id_generator.generate(
+                            translation_domain.ContentType.SOLUTION),
+                        'html': '<p>This is a solution.</p>',
+                    },
+                }
+                hints_list = [
+                    state_domain.Hint(
+                        state_domain.SubtitledHtml(
+                            content_id_generator.generate(
+                                translation_domain.ContentType.HINT),
+                            '<p>This is a hint.</p>')),
+                ]
+                # Ruling out the possibility of None for mypy type checking,
+                # because we above we are already updating the value
```

Comment:
typo, please proofread PRs

Reply:
Also, this comment and one below - try to make comments concise. Comments should point out something extraordinary, and the longer the comment the more attention it attracts. These two comments are not very important, try to make it 1, at most 2 lines.

Reply:
Done!